Desktop/components/JASP/Controls/ComponentsList.qml

```diff
@@ -68,18 +150,20 @@ JASPGridControl
 		FocusScope
 		{
 			id		: itemWrapper
-			height	: rowComponentsItem.height
+			height	: rowComponentItem ? rowComponentItem.height : 0
 			width	: componentsList.itemGrid.width
 
-			property bool isDeletable: addItemManually && (!model.type || model.type.includes("deletable"))
+			property var	rowComponentItem	: model.rowComponent
+			property bool	isDeletable			: addItemManually && (!model.type || model.type.includes("deletable"))
 
-			RowComponents
+			Component.onCompleted:
 			{
-				id						: rowComponentsItem
-				anchors.verticalCenter	: parent.verticalCenter
-				anchors.left			: parent.left
-				spacing					: componentsList.columnSpacing
-				controls				: model.rowComponents
+				if (rowComponentItem)
+				{
+					rowComponentItem.parent = itemWrapper;
```

Comment:
Shouldn't these settings be set on the `Repeater` at line 120 to 125?
Well, I have more questions about this component but I already had those if I look at the old code so nvm.